pkgmgmtutils/utils_test.go

```diff
@@ -1,160 +0,0 @@
-package pkgmgmtutils
-
-import (
-	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"regexp"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("package management utils", func() {
```

Comment:
we just deleted all these tests without any replacement for them after the refactor?

Reply:
There are maybe two that need replacing. The vast majority of this file is either covered already by my new tests or is dead code paths.